slides/08-Platform-Hub/1010-WhatIsIt.md

```diff
@@ -0,0 +1,8 @@
+## What is the Platform Hub?
+
+- Accessible webpage for managing projects and teams
```

Comment:
Have James Buller and his team validated that statement?

Reply:
We can't assert that the hub is fully "accessible" at the moment, so best to rephrase this.